deployment/templates/fluentd.yaml

```diff
@@ -84,6 +84,9 @@ objects:
             - name: dockerhostname
               mountPath: /etc/docker-hostname
               readOnly: true
+            - name: throttle-settings
```

Comment:
Hmm, this is probably redundant since the readme gives the command to mount the secret where this volume mount currently is.  I'll update the yaml to remove this mount and the emptyDir volume definition.  The Dockerfile creates the structure and empty file to be overwritten, so this shouldn't be needed.